microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -25,7 +25,7 @@ This article is for WebView2 developers.
 <!-- ====================================================================== -->
 ## Optional diagnostic data
 
-WebView2 is a Windows component.  The data collection consent for WebView2 is governed by the **Settings > Privacy & security > Diagnostics & feedback > Diagnostic data** setting on Windows 10 and Windows 11 as a centralized switch.  Similar to the Windows APIs, WebView2 developers have no control over the Windows **Diagnostic data** setting.
+WebView2 is a Windows component and thus follows the Windows diagnostic data collection practices. For more information see [Windows diagnostic data](https://learn.microsoft.com/en-us/windows/privacy/configure-windows-diagnostic-data-in-your-organization).  The data collection consent for WebView2 is governed by the **Settings > Privacy & security > Diagnostics & feedback > Diagnostic data** setting on Windows 10 and Windows 11 as a centralized switch.

```

Comment:
```suggestion
WebView2 is a Windows component and thus follows the Windows diagnostic data collection practices. For more information, see [Windows diagnostic data](/windows/privacy/configure-windows-diagnostic-data-in-your-organization).  The data collection consent for WebView2 is governed by the **Settings > Privacy & security > Diagnostics & feedback > Diagnostic data** setting on Windows 10 and Windows 11 as a centralized switch.
```

https://learn.microsoft.com/en-us/windows/privacy/configure-windows-diagnostic-data-in-your-organization